### DISCUSSION TITLE
does this work on non TT devices

### DISCUSSION THREAD

**Discussion by bigchungus2222** [devcompl/TTController#208] [General]
Hi,

I've changed computer and no longer have one with thermalake however your script for TT controller was the best and easiest RGB controller I've used.

does it work on non thermalake devices?

Thanks,

Alex

**Comment by MoshiMoshi0** [comment]
Nope, for fans I would recommend "FanControl", for RGB there isnt anything good tbh, "OpenRGB" is the least bad imo.
